api/core/tools/provider/builtin/json_process/tools/insert.yaml

```diff
@@ -0,0 +1,86 @@
+identity:
+  name: json_insert
+  author: Mingwei Zhang
+  label:
+    en_US: JSON Insert
+    zh_Hans: JSON 插入
+    pt_BR: JSON Insert
+description:
+  human:
+    en_US: A tool for inserting JSON content
+    zh_Hans: 一个插入 JSON 内容的工具
+    pt_BR: A tool for inserting JSON content
+  llm: A tool for inserting JSON content
+parameters:
+  - name: content
+    type: string
+    required: true
+    label:
+      en_US: JSON content
+      zh_Hans: JSON 内容
+      pt_BR: JSON content
+    human_description:
+      en_US: JSON content
+      zh_Hans: JSON 内容
+      pt_BR: JSON content
+    form: llm
```

Comment:
llm_description is needed, otherwise it make no sense for Agent

Reply:
Thank you for your suggestion. The llm_descriptions have been added.